src/utils/dispute-utils.js

```diff
@@ -44,6 +44,20 @@ export function transformRoundDataAttributes(round) {
   }
 }
 
+function transformDisputableDataAttributes(dispute) {
+  if (!dispute.disputable) {
+    return null
+  }
+
+  const { disputable } = dispute
+
+  return {
+    ...disputable,
+    actionContext: toUtf8String(disputable.actionContext || '0x'),
```

Comment:
Right, we weren't able to decode the string from the subgraph? Was wondering if we should also use the `true` flag like we do in `parseMetadata()`?

Reply:
@facuspagnuolo just updated the subgraph where it's decoded from the subgraph, so just updated the PR.

See https://github.com/aragon/court-subgraph/pull/75